### ISSUE TITLE
Can I disable italics?

### ISSUE THREAD

**Issue by NDelt** [Binaryify/OneDark-Pro#328]
Currently, some variables and comments are displayed with italics.
How to I disable this? I've modified `OneDark-Pro.json` file but it is initialized again.

**Comment by Binaryify** [comment]
you should modified it on setting.json

**Comment by Binaryify** [comment]
will support close italics in next version

**Comment by Binaryify** [comment]
just support this in v3.0

**Comment by NDelt** [comment]
@Binaryify Thanks! It works well :)

**Comment by estebanborai** [comment]
Hi @Binaryify ! Thanks for such a great extension/vscode theme!

I'm looking into disable italics at all when using this theme, but even when I enabled the setting above (https://github.com/Binaryify/OneDark-Pro/issues/328#issuecomment-554817179), I'm still having italics in my files.

For example JSX attributes.

![onedarkpro-italic](https://user-images.githubusercontent.com/34756077/81504407-6474b380-92bf-11ea-84c0-5218e5ef54ba.png)

I added the rule as follows:
`  "oneDarkPro.italic": false`

And is also checked in the `Settings` file.

Thanks in advance!
